clients/admin-ui/src/pages/management/domains.tsx

```diff
@@ -189,6 +257,42 @@ const CORSConfigurationPage: NextPage = () => {
             )}
           </FormSection>
         </Box>
+        <Box maxW="600px" marginY={3}>
```

Comment:
Let's use padding here instead to avoid margin collapse, this will space out the two forms more evenly
```suggestion
        <Box maxW="600px" paddingY={3}>
```

Reply:
Good call!